ui/pages/confirmations/hooks/useCurrentConfirmation.test.ts

```diff
@@ -56,7 +56,7 @@ describe('useCurrentConfirmation', () => {
     );
 
     expect(result.current.currentConfirmation).toBe(
-      mockState.metamask.unapprovedPersonalMsgs['1'],
+      mockState.metamask.unapprovedPersonalMsgs['2'],
```

Comment:
I don't understand the changes to this file. Are they necessary? This looks like it is passing on `develop`

Reply:
Hmmm, it was breaking on CI and locally. I can revert again and see what happens 🤷‍♂️